app/controllers/api/v1/brands_controller.rb

```diff
@@ -1,8 +1,32 @@
 class Api::V1::BrandsController < Api::V1::BaseController
-  before_action :get_brand, only: [:show]
+  before_action :get_brand, only: [:show, :update_account_email]
   before_action :ensure_user_can_perform_action, only: [:show]
 
   def show
     render json: @brand, serializer: BrandSerializer
   end
+
+  def update_account_info
+    binding.pry
+
```

Comment:
no clue why params are not showing here now [
<img width="870" alt="screen shot 2016-10-17 at 10 11 28 pm" src="https://cloud.githubusercontent.com/assets/17052014/19461941/e6475a80-94b6-11e6-9a00-9da0247ddd54.png">
](url)